platform/darwin/scripts/generate-style-code.js

```diff
@@ -632,6 +633,7 @@ const layers = _(spec.layer.type.values).map((value, layerType) => {
     }, []);
 
     const paintProperties = Object.keys(spec[`paint_${layerType}`]).reduce((memo, name) => {
+        if (name === 'heatmap-color') return memo;
```

Comment:
TODO: remove this before merging

Reply:
@kkaefer we need to overhaul the code generation across all platforms to support expressions in default values — not sure how big of a push that is after expressions support landed for Darwin a few days ago, but probably very non-trivial. @anandthakker suggested side-stepping this for now (and not supporting runtime color change initially) — Anand, should we reconsider this? How much of a push that would be?

Reply:
Now that expressions support has landed, I think we should, indeed, remove this and support runtime styling for `heatmap-color`.  It probably won't be trivial, but I'm hoping that it won't be too bad.